templates/consumption/pdf/default.html.twig

```diff
@@ -0,0 +1,8 @@
+<html>
+<body>
+<h1>
+Uw overzicht
+</h1>
+    {% include 'consumption/partial/consumptionstable.html.twig' %}
```

Comment:
Werkt de tailwind styling in de PDF?

Reply:
nee. Pdf wordt gewoon zo simpel mogelijk getoond zonder al te veel styling. Printscreen heb ik naar de groep verstuurd